sdk/communication/azure-communication-identity/src/test/resources/session-records/revokeTokenWithResponseWithNullUser.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
what is this file?

Reply:
This is a recorded response for a test that throws a NullExceptionError and so it does not make a server side call (hence why it is empty). Even though it is empty, we still need to in order for playback tests to run properly